src/compute-client/src/command.rs

```diff
@@ -673,8 +673,12 @@ impl RustType<ProtoDataflowDescription>
             objects_to_build: proto.objects_to_build.into_rust()?,
             index_exports: proto.index_exports.into_rust()?,
             sink_exports: proto.sink_exports.into_rust()?,
-            as_of: proto.as_of.map(Into::into),
-            until: proto.until.map(Into::into).unwrap_or_else(Antichain::new),
+            as_of: proto.as_of.map(|x| x.into_rust()).transpose()?,
```

Comment:
these two were the only ones that required any thought, the rest were purely mechanical. calling this out for particular attention in review